src/libs/actions/Report.js

```diff
@@ -960,9 +959,8 @@ function editReportComment(reportID, originalReportAction, textForNewComment) {
 
     const parameters = {
         reportID,
-        sequenceNumber,
```

Comment:
@neil-marcellini @luacmartins I'm assuming here that we're making sequenceNumber an optional parameter for `EditReportComment`